subprojects/docs/src/docs/userguide/authoring-builds/authoring_maintainable_build_scripts.adoc

```diff
@@ -69,29 +69,30 @@ include::sample[dir="snippets/bestPractices/conditionalLogic-do/groovy", files="
 
 Use of Gradle internal APIs in plugins and build scripts has the potential to break builds when either Gradle or plugins change.
 
-The following packages are listed in the link:https://github.com/gradle/gradle/blob/180b9d3fa84b91768364c603380e82947437eda1/buildSrc/subprojects/configuration/src/main/kotlin/org/gradle/gradlebuild/public-api.kt[Gradle public API definition], with the exception of any subpackage with `internal` in the name:
-
+The following packages are listed in the
+link:https://github.com/gradle/gradle/blob/6961503e70aa634653e71413224831fd3fa3ec2d/build-logic/basics/src/main/kotlin/gradlebuild/basics/PublicApi.kt[Gradle public API definition]
+and the
+link:https://github.com/gradle/gradle/blob/6961503e70aa634653e71413224831fd3fa3ec2d/build-logic/basics/src/main/kotlin/gradlebuild/basics/PublicKotlinDslApi.kt[Kotlin DSL API definition], except any subpackage with `internal` in the name:
+// If this list is modified due changes in the public API definitions, update the permalinks too
```

Comment:
🤔 A further improvement could be here to actually list _packages_ (`org.gradle.trallala`) instead of paths (`org/gradle/trallala`).

Reply:
I think that makes a lot of sense from the docs reader perspective. Done!